# LLD 16 causes the qt version symbols to not be exported by the shared library. Fix this so that other things will link correctly.

diff -up qtbase-everywhere-src-5.15.8/mkspecs/features/qt_module.prf.omv~ qtbase-everywhere-src-5.15.8/mkspecs/features/qt_module.prf
--- qtbase-everywhere-src-5.15.8/mkspecs/features/qt_module.prf.omv~	2023-02-25 15:50:49.584159791 +0100
+++ qtbase-everywhere-src-5.15.8/mkspecs/features/qt_module.prf	2023-02-25 15:50:10.780158607 +0100
@@ -236,12 +236,14 @@ android: CONFIG += qt_android_deps no_li
         isEmpty(QT_NAMESPACE): tag_symbol = qt_version_tag
         else:                  tag_symbol = qt_version_tag_$$QT_NAMESPACE
 
+        contains(TARGET, .*QtCore.*) {
         for(i, 0..$$QT_MINOR_VERSION) {
             previous = $$current
             current = Qt_$${QT_MAJOR_VERSION}.$$i
             equals(i, $$QT_MINOR_VERSION): verscript_content += "$$current { $$tag_symbol; } $$previous;"
             else:                          verscript_content += "$$current {} $$previous;"
         }
+        }
 
         # Add a post-processing step to replace the @FILE:filename@
         verscript_in = $${verscript}.in
